admin/test_langs.php

```diff
@@ -100,7 +100,7 @@
  * @return void
  */
 function checkplugins( $p_path ) {
-	$t_path = rtrim( $p_path, DIRECTORY_SEPARATOR ) . DIRECTORY_SEPARATOR;
+	$t_path = rtrim( $p_path, DIRECTORY_SEPARATOR ) . '/';
```

Comment:
You still have directory separator used. The trim and concat should use the same construct.

Reply:
> You still have directory separator used.

That's intentional and required in this case, because $p_path is coming from FileSystemIterator so may contain `/` or `\` depending on the underlying OS.

> The trim and concat should use the same construct.

This is actually not necessary, as PHP internal functions correctly interpret `/` regardless of OS. So not using DIRECTORY_SEPARATOR saves keystrokes, reduces the need for concatenation (e.g. `'/lang/'` vs `DIRECTORY_SEPARATOR . 'lang' . DIRECTORY_SEPARATOR`, and ultimately makes the code easier to read.